Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:3.11-slim-bookworm
+FROM tangowithfoxtrot/bw-cli as builder
 
-RUN apt update && apt install -y npm && \
-    npm i -g @bitwarden/cli && \
-    apt purge -y npm
+FROM python:3.11.0-slim-bullseye
+
+COPY --from=builder /usr/local/bin/bw /bin/bw
```

Comment:
What about just downloading latest binary from [here](https://bitwarden.com/help/cli/#download-and-install) and use original base docker image. I would really like to avoid using unknown/less known docker images.

Reply:
That URL can change... I will create a script to download it from Github instead, like the image I'm basing from suggest 👍🏾

Reply:
Done! I went with your recommendation and kept the same size benefits. Let me know if other changes are needed!